apps/afisha/admin/events.py

```diff
@@ -113,11 +114,15 @@ def status(self, obj):
         def icon(status, lable):
             return mark_safe(f"<img src='/static/admin/img/{status}.svg' title='{lable}'/>")
 
-        if obj.date_time.date() > date_now:
-            return icon("upcoming", "Предстоящее")
-        elif obj.date_time.date() < date_now:
-            return icon("past", "Прошедшее")
-        return icon("today", "Cегодняшнее")
+        if obj.date_time:
+            if obj.is_archived:
```

Comment:
Эти два условия можно объединить, так будет проще и читаемее
``
```suggestion
            if obj.is_archived or obj.date_time.date() < date_now::
```

Reply:
А можно даже так сделать, код намного станет проще
```suggestion
            if not obj.date_time or obj.is_archived or obj.date_time.date() < date_now:
```